.travis.yml

```diff
@@ -56,7 +56,6 @@ notifications:
           - http://julia.mit.edu:8000/travis-hook
 before_install:
     - make check-whitespace
-    - JULIA_SYSIMG_BUILD_FLAGS="--output-ji ../usr/lib/julia/sys.ji"
```

Comment:
bootstrapping is NOT testing the case where you have a ji file but not a dll file, which systems without linkers need to continue testing